engine/src/lin_time.cpp

```diff
@@ -113,15 +113,18 @@ void setTimeCompression( float tc )
 bool toggle_pause()
 {
     static bool paused = false;
+    fprintf(stderr, "toggle_pause() called in lin_time.cpp\n");
     if (paused)
     {
+        fprintf(stderr, "toggle_pause() in lin_time.cpp: Resuming (unpausing)\n");
         setTimeCompression(1);
         paused = false;
     }
     else
     {
-        setTimeCompression(.0000001);
-	paused = true;
+        fprintf(stderr, "toggle_pause() in lin_time.cpp: Pausing\n");
+        setTimeCompression(.00001);
```

Comment:
This is the crux of the fix here. Apparently, when time compression is set to a very low value, it has the effect of slowing the entire game down -- including processing keystrokes. The game "pauses" by slowing itself down drastically. Then it "resumes", or unpauses, by returning to normal speed.

The problem was the time compression coefficient. It was being set so slow that the "resume" key never registered. Neither did anything else. Or at least, not for probably hours and hours.

So I simply changed the coefficient by a couple of orders of magnitude, and problem solved. That was the fix, best I can tell.

Reply:
It would be good to have a comment here about that so we don't do that again.

We might even need to make that CPU dependent...not in this resolution though.

Reply:
> It would be good to have a comment here about that so we don't do that again.
> 
> We might even need to make that CPU dependent...not in this resolution though.

@BenjamenMeyer Comment added.